src/cmdc_tools/datasets/official/OK/counties/tulsa.py

```diff
@@ -0,0 +1,79 @@
+import pandas as pd
+import textwrap
+
+from ... import ArcGIS
+from .... import DatasetBaseNoDate
+
+
+class OKTulsa(DatasetBaseNoDate, ArcGIS):
+    """
+    Tulsa, Oklahoma publishes their county level data in a dashboard that can
+    be found at:
+
+    https://tcchd.maps.arcgis.com/apps/opsdashboard/index.html#/ebb119cd215b4c57933b7fbe477e7c30
+
+    This retrieves the data from the ArcGIS API
+    """
+
+    ARCGIS_ID = "yHSoWow4TMdKqS8y"
+    source = (
+        "https://tcchd.maps.arcgis.com/apps/opsdashboard/index.html#/"
+        "ebb119cd215b4c57933b7fbe477e7c30"
+    )
+    state_fips = 40
+
+    def __init__(self, params=None):
```

Comment:
This init method seems to do nothing. It can be deleted.

Reply:
Done.